addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -17,7 +17,7 @@
 
     <!-- Link Dialog (allows to choose a style and content for a link on the page) -->
     <div t-name="web_editor.LinkDialog">
-        <Dialog size="'xl'">
+        <Dialog size="'xl'" title="'Insert a Link / Button'">
```

Comment:
I don't think this will be translated unless you do `env._t('Insert a Link / Button')`.

Can you give it a try?

Reply:
Hello @awa-odoo 

Changes have been made.

Thanks!